File: async-scala-client/src/main/scala/io/swagger/client/model/SefazInvoiceStatus.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class SefazInvoiceStatus (
  code: Option[String],
desc: Option[String],
protocol: Option[String],
rec: Option[String],
date: Option[DateTime],
environment: Option[String],  // - &#39;1&#39; # Ambiente de Produção - &#39;2&#39; # Ambiente de Homologação 
appVersion: Option[String]
)

File: javascript-closure-angular-client/API/Client/SefazInvoiceStatus.js
goog.provide('API.Client.SefazInvoiceStatus');

/**
 * @record
 */
API.Client.SefazInvoiceStatus = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.SefazInvoiceStatus.prototype.code;

/**
 * @type {!string}
 * @export
 */
API.Client.SefazInvoiceStatus.prototype.desc;

/**
 * @type {!string}
 * @export
 */
API.Client.SefazInvoiceStatus.prototype.protocol;

/**
 * @type {!string}
 * @export
 */
API.Client.SefazInvoiceStatus.prototype.rec;

/**
 * @type {!Date}
 * @export
 */
API.Client.SefazInvoiceStatus.prototype.date;

/**
 * - '1' # Ambiente de Produção - '2' # Ambiente de Homologação 
 * @type {!string}
 * @export
 */
API.Client.SefazInvoiceStatus.prototype.environment;

/**
 * @type {!string}
 * @export
 */
API.Client.SefazInvoiceStatus.prototype.appVersion;

/** @enum {string} */
API.Client.SefazInvoiceStatus.EnvironmentEnum = { 
  1: '1',
  2: '2',
}
